sync-version/action.yaml

```diff
@@ -36,11 +69,21 @@ runs:
     - name: Commit changes
       if: steps.git-check.outputs.modified == 'true'
       shell: bash
+      with:
+        yaml_path: ${{ steps.yaml-path.outputs.yaml_path }}
+        prev_version: ${{ steps.yaml-path.outputs.prev_version }}
       env:
-        SNAP_NAME: ${{ github.event.repository.name }}
+        SNAP_NAME: ${{ inputs.snap-name || github.event.repository.name }}
```

Comment:
We definitely can, in fact, we should do that way. I never thought about that seeing the fact that currently names are taken from the github repo name. I'll pass the name from the `yaml-path` step.

> I want to make sure our inputs are air-tight since once we commit this, we will never be allowed to change it.

BTW, curious to know why so?